sdk/servicenetworking/Azure.ResourceManager.ServiceNetworking/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure ServiceNetworking management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Please add the description here

Reply:
I've added a description for the time being, but I need to run this by our PM before we release this.